subprojects/gradle/buildchecks/build.gradle.kts

```diff
@@ -18,7 +18,6 @@ dependencies {
     implementation(project(":gradle:kotlin-dsl-support"))
     implementation(Dependencies.Gradle.kotlinPlugin)
     implementation(Dependencies.funktionaleTry)
-    implementation(Dependencies.Gradle.androidPlugin)
```

Comment:
How did you find this unnecessary dependency? 
The `dependency-analysis` plugin?

Reply:
nope, manual check